lib/script/script.js

```diff
@@ -187,6 +187,34 @@ class Script {
     return new this().fromArray(code);
   }
 
+  /**
+   * Inject P2WSH script from redeem script.
+   * @private
+   * @param {Script} redeemScript
+   * @returns {Script}
+   */
+
+  fromRedeemScript(script) {
```

Comment:
We already have these methods:

example:

```
const script = Script.fromMultisig(2, 3,
  [
    ring1.publicKey,
    ring2.publicKey,
    ring3.publicKey
  ]);

Script.fromProgram(0. script.sha256())


```

We wouldn't want to hard-code a SegWit version with a `OP_0` push, either (taproot is coming!)